src/components/AoFileUpload.vue

```diff
@@ -80,6 +84,10 @@ export default {
   },
 
   computed: {
+    uniqId () {
+      return this.name ? uniqueId(`${this.name}_`) : uniqueId()
```

Comment:
Maybe add `blaze_input` as a prefix in both cases to reduce the likelyhood of collision with something else. Could also move uniqId to component_utilities.js or something to make this a bit less repetitive.